examples/create_alert.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: localhost
+
+  tasks:
+  - name: Create an alert in ManageIQ
+    manageiq_alert:
+      miq_url: http://miq.example.com
+      miq_username: admin
+      miq_password: secret
+      miq_verify_ssl: false
+      description: Test Alert 01
+      options:
+        notifications:
+          delay_next_evaluation: 0
+          evm_event: {}
+      entity: node
+      expression:
+        eval_method: dwh_generic
+        mode: internal
+      expression_type: hash
+      enabled: true
+      state: present
```

Comment:
Daniel can you make sure the alert you showed me today can be created using this example?